source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -56,7 +56,7 @@ Settings
           - :ref:`<source-usage-example-custom-pipeline>`
           - :ref:`<source-usage-example-multiple-sources>`
 
-       | **Default**: ``[]``
+       | **Default**: ``"[]"``
        | **Accepted Values**: Valid aggregation pipeline stage
```

Comment:
suggestion: under accepted values, we could indicate that the pipeline should be a string

Reply:
After some investigation, it seems like this doesn't exactly have to be a string since there are some automatic conversions happening behind the scenes. I'm going to leave the Accepted Values section as is for now because of that.